src/Microsoft.Extensions.CultureInfoGenerator.Sources/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
```

Comment:
Hmm, why is the generator a "Sources" package? Shouldn't it be a regular old console app package that can be run?